### PR TITLE
Adjustments to Avail documentation

### PR DESCRIPTION
**Pull request by staking4all**
Adjustments that include

- Update node version
- Update monitoring for validators that have external Prometheus
- Add fast upgrade process
- Update grafana dashboard to read from Avail repo

Thanks

### CODE REVIEW

**Review by delroybosco — APPROVED** [review]

File: docs/join-the-network/run-avail/validator-ops/0030-avail-upgrade-validator-node.md [hunk 1]
```diff
@@ -13,9 +13,46 @@ slug: avail-upgrade-validator-node
...
+While a rapid upgrade is possible, it carries inherent risks. For instance, if you proceed with the upgrade and encounter issues such as database corruption, there's a chance of prolonged downtime. This could lead to your node being removed from the active validator set due to unresponsiveness.
+
+The fast upgrade steps are:
+
+- Stop the Avail node.
+```
+sudo systemctl stop availd.service
+```
+
+- Locate your Avail binary, create a backup of the current binary, and then uninstall the existing binary by deleting the binary. Proceed to download the most recent binary announced in Discord, which will replace the previous version.
```
Comment (vthunder) [inline docs/join-the-network/run-avail/validator-ops/0030-avail-upgrade-validator-node.md:29]: I think this should just link over to the installation page which will always have the official download links:

https://availproject.github.io/join-the-network/run-avail/full-node-setup

And say that the binary needs to be downloaded and placed in the same folder as where the old one was and started back up in the same way as before.

File: docs/join-the-network/run-avail/validator-ops/0030-avail-upgrade-validator-node.md.save
Comment (vthunder) [inline docs/join-the-network/run-avail/validator-ops/0030-avail-upgrade-validator-node.md.save:1]: can we remove this file? the old version will be in the git history, we don't need another copy

**Review by vthunder — APPROVED** [review]
Thank you! Left a few comments, please address those, and then merge (no need for another review).

File: docs/join-the-network/run-avail/validator-ops/0030-avail-upgrade-validator-node.md [hunk 2]
```diff
@@ -13,9 +13,46 @@ slug: avail-upgrade-validator-node
 ---
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-## Upgrade Process
+When upgrading Avail nodes there is two options for upgrading. Below two sections provide details on each upgrade process. You only need to use one of the below upgrade processes.
```
[the remark this answers is not in the document]
Comment (staking4all) [reply docs/join-the-network/run-avail/validator-ops/0030-avail-upgrade-validator-node.md]: Thanks, have done these changes accordingly

File: docs/join-the-network/run-avail/validator-ops/0030-avail-upgrade-validator-node.md [hunk 3]
```diff
@@ -13,9 +13,46 @@ slug: avail-upgrade-validator-node
...
-Upgrading a Avail node involves a careful process to ensure a smooth transition without disruption to the network. Here's a step-by-step guide on how to upgrade a Avail node, 
+## Fast Upgrade
+
+While a rapid upgrade is possible, it carries inherent risks. For instance, if you proceed with the upgrade and encounter issues such as database corruption, there's a chance of prolonged downtime. This could lead to your node being removed from the active validator set due to unresponsiveness.
+
+The fast upgrade steps are:
+
+- Stop the Avail node.
+```
+sudo systemctl stop availd.service
```
[the remark this answers is not in the document]
Comment (staking4all) [reply docs/join-the-network/run-avail/validator-ops/0030-avail-upgrade-validator-node.md:28]: we added the system service part in the validator section as we found less experienced validators were struggling. Can be found here:
https://availproject.github.io/join-the-network/run-avail/validator-node-setup

So this continues from validator setup. But you are correct that we can not just assume. So I updated to reference validator setup

_To provide an example, assuming your existing binary is located at `/home/avail/avail-node/` and is named `data-avail`, and you used the [validator](/join-the-network/run-avail/validator) setup guidelines while
obtaining a pre-built binary from the Avail GitHub repository, proceed as outlined below._

But in theory the fast upgrade steps could be used on non validator nodes like rpc, archive, etc. So added a few more words in each section highlighting what is most appropriate.

File: docs/join-the-network/run-avail/validator-ops/0030-avail-upgrade-validator-node.md (hunk 1 above)
Comment (staking4all) [reply docs/join-the-network/run-avail/validator-ops/0030-avail-upgrade-validator-node.md]: updated to link to validator installation part ad most appropriate
https://availproject.github.io/join-the-network/run-avail/validator-node-setup

Left _announcement in discord_ part as the instruction has been to only install new binaries once testing is done and confirmed. For example testnet is running 1.6.2, there are two other binaries available after this version but all validators requested to stay on 1.6.2

File: docs/join-the-network/run-avail/validator-ops/0030-avail-upgrade-validator-node.md.save
Comment (staking4all) [reply docs/join-the-network/run-avail/validator-ops/0030-avail-upgrade-validator-node.md.save:1]: removed, thanks. was created by accident from editor

**Review by DannyS03 — APPROVED** [review]
Checked that most recent suggestions were applied, lgtm.

**Pull request merged**